app/assets/javascripts/models/page/search_page.js.coffee

```diff
@@ -50,7 +54,11 @@ ns.SearchPage = do (ko
     $('.launch-customize-modal').click ->
       $('#customizeDataModal').modal('show')
 
-    preferences.ready.done(-> initModal()) if !window.edscportal
+    reverb = ["http://echo-reverb-rails.dev", "https://testbed.echo.nasa.gov", "https://api-test.echo.nasa.gov", "https://testbed.echo.nasa.gov", "https://reverb.echo.nasa.gov"]
+    if $.inArray(document.referrer, reverb) != -1 && Cookies.get('ReadyForReverbRetirement') != 'true'
+      $('#reverbRetirementModal').modal('show')
+    else
```

Comment:
Use a separate ```if``` to separate the redirect from the tour (or anything).

Reply:
My concern is that the two modals would compete in some unhealthy way.  What if the site tour modal shows up on *top* of the reverb modal?  By doing an if/then, the site tour modal is effectively momentarily suppressed in the case of a Reverb referral.

Reply:
The redirect modal will show up before the tour modal since it is loaded right after dom loads. At this time, none of ajax calls has been fired yet.